packages/babel-preset-default/index.js

```diff
@@ -75,21 +75,10 @@ module.exports = ( api ) => {
 		],
 		plugins: [
 			require.resolve( '@wordpress/warning/babel-plugin' ),
-			[
-				require.resolve( '@wordpress/babel-plugin-import-jsx-pragma' ),
```

Comment:
We need to deprecate this and ensure a breaking change release for babel-preset-default and scripts packages.